.dockerignore

```
@@ -0,0 +1,5 @@
+.env
+.venv
+.vscode
+Dockerfile
+.git
```

This is a good minimal list, I'm a fan of adding any caching folders here too as I've been caught out by those in the past, e.g.:
```suggestion
.git
**/__pycache__
**/.pytest_cache
```
(Note the use of `**/`, which means to consider folders of those names at any level, not just at the root directory level. This is unlike `.gitignore`, which automatically takes that approach!)